h2o-algos/src/main/java/hex/schemas/StackedEnsembleV99.java

```diff
@@ -51,15 +51,13 @@
                   " The 'Stratified' option will stratify the folds based on the response variable, for classification problems.")
     public Model.Parameters.FoldAssignmentScheme metalearner_fold_assignment;
 
-    // TODO: Add metalearner_fold_column
-    // also requires: import water.api.schemas3.FrameV3;
-    /*
+    // For ensemble metalearner cross-validation
     @API(level = API.Level.secondary, direction = API.Direction.INOUT, gridable = true,
             is_member_of_frames = {"training_frame"},
-            is_mutually_exclusive_with = {"ignored_columns", "response_column", "weights_column", "offset_column"},
-            help = "Column with cross-validation fold index assignment per observation.")
+            //is_mutually_exclusive_with = {"ignored_columns", "response_column", "weights_column", "offset_column"},
+            is_mutually_exclusive_with = {"response_column"},
```

Comment:
I should check about adding `ignored_columns` back in for now... Stacked Ensemble doesn't have weights or offset support yet, so we don't need those.

Reply:
Added `ignored_columns` back in.